.jenkins

```diff
@@ -11,6 +11,9 @@ pipeline {
         BENCHMARK_COLOR = 'no'
         BOOST_TEST_COLOR_OUTPUT = 'no'
         CTEST_OPTIONS = '--timeout 180 --no-compress-output -T Test --test-output-size-passed=65536 --test-output-size-failed=1048576'
+        OMP_NUM_THREADS = 8
+        OMP_PLACES = 'threads'
```

Comment:
Not sure about this one and whether `OMP_PLACES='cores'` is a better option. But it does not matter much.

Reply:
I just blindly followed the recommendation from the warning issued by Kokkos